frontends/infinite-corridor/src/scss/learning-resources.scss

```diff
@@ -9,7 +9,7 @@ $cardListSpacer: 1.5em;
 
 .ic-resource-card {
   &.MuiCard-root {
-    color: theme.$color-gray-5;
+    color: theme.$font-color-default;
```

Comment:
I was originally modeling the card styles after `/learn` and the initial infinite designs, which both use black fonts.

We want to match the open-discussions cards, which use the dark blue font. (Which is `theme.$font-color-default`)

Similar reason for the semibold -> bold change below.